core/backend/src/Element.ts

```diff
@@ -336,38 +336,55 @@ export class Element extends Entity {
-   * This is important for cloning operations but can be useful in other situations as well.
-   * @param predecessorIds The Id64Set to populate with predecessor Ids.
-   * @note In order to clone/transform an element, all predecessor elements must have been previously cloned and remapped within the [IModelCloneContext]($backend).
-   * @note This should be overridden (with `super` called) at each level the class hierarchy that introduces predecessors.
-   * @see getPredecessorIds
+  /** Collect the Ids of this element's *references* at this level of the class hierarchy.
+   * @deprecated use collectReferenceIds instead, the use of the term *predecessors* was confusing and became inaccurate when the transformer could handle cycles
    * @beta
    */
   protected collectPredecessorIds(predecessorIds: Id64Set): void {
-    predecessorIds.add(this.model); // The modeledElement is a predecessor
+    return this.collectReferenceIds(predecessorIds);
+  }
+
+  /** Collect the Ids of this entity's *references* at this level of the class hierarchy.
+   * A *reference* is any entity referenced by this entity's EC Data
+   * This is important for cloning operations but can be useful in other situations as well.
+   * @param referenceIds The Id64Set to populate with reference Ids.
+   * @note In order to clone/transform an entity, all referenced elements must have been previously cloned and remapped within the [IModelCloneContext]($backend).
+   * @note This should be overridden (with `super` called) at each level the class hierarchy that introduces references.
+   * @see getReferenceIds
+   * @beta
+   */
+  protected collectReferenceIds(referenceIds: Id64Set): void {
+    referenceIds.add(this.model); // The modeledElement is a reference
     if (this.code.scope && Id64.isValidId64(this.code.scope))
-      predecessorIds.add(this.code.scope); // The element that scopes the code is a predecessor
+      referenceIds.add(this.code.scope); // The element that scopes the code is a reference
     if (this.parent)
-      predecessorIds.add(this.parent.id); // A parent element is a predecessor
+      referenceIds.add(this.parent.id); // A parent element is a reference
   }
 
-  /** Get the Ids of this element's *predecessors*. A *predecessor* is an element that had to be inserted before this element could have been inserted.
+  /** Get the Ids of this element's *references*. A *reference* is an element that this element references.
    * This is important for cloning operations but can be useful in other situations as well.
-   * @see collectPredecessorIds
    * @beta
+   * @deprecated
    */
   public getPredecessorIds(): Id64Set {
-    const predecessorIds = new Set<Id64String>();
-    this.collectPredecessorIds(predecessorIds);
-    return predecessorIds;
+    return this.getReferenceIds();
+  }
+
+  /** Get the Ids of this element's *references*. A *reference* is an element that this element references.
```

Comment:
"A *reference* is an element that this element references." is rather tautological - can you be more descriptive?

Reply:
you're right... I kept to the original desc format way too much, will fix

Reply:
how about now?